examples/hovorod-cpu/README.md

```diff
@@ -0,0 +1,21 @@
+# Horovod CPU-Only Case
+
+This example shows how to run a cpu-only mpijob.
+
+## How to Build Image
+
+Please download Horovod cpu only dockerfile and then build the image as follows:
+
+```bash
+curl -O https://raw.githubusercontent.com/horovod/horovod/master/Dockerfile.cpu
```

Comment:
Can we copy the Dockerfile to this repo?

Reply:
Yes, I have copied it.